packages/react-data-grid-addons/src/toolbars/GroupedColumnsPanel.js

```diff
@@ -31,7 +31,10 @@ class GroupedColumnsPanel extends Component {
 
   renderGroupedColumns() {
     return this.props.groupBy.map(c => {
-      return (<GroupedColumnButton key={c} name={c} onColumnGroupDeleted={this.props.onColumnGroupDeleted}/>);
```

Comment:
Would rather not repeat the jsx assignments, it can be:

```
const groupedColumnButtonProps = {
   columnKey: c,
   name: c,
   onColumnGroupDeleted: this.props.onColumnGroupDeleted,
   key: typeof c === 'string' ? c : c.key
}

return <GroupedColumnButton {...groupedColumnButtonProps} />;
```